command/jsonconfig/expression.go

```diff
@@ -0,0 +1,126 @@
+package jsonconfig
+
+import (
+	"encoding/json"
+
+	"github.com/hashicorp/hcl2/hcl"
+	"github.com/hashicorp/hcl2/hcldec"
+	"github.com/hashicorp/terraform/configs/configschema"
+	"github.com/hashicorp/terraform/lang"
+	"github.com/zclconf/go-cty/cty"
+	ctyjson "github.com/zclconf/go-cty/cty/json"
+)
+
+// // expressions is used to represent the entire content of a block. Attribute
+// // arguments are mapped directly with the attribute name as key and an
+// // <expression-representation> as value.
+// type expressions struct {
```

Comment:
Looks like this is a leftover older version of `type expressions` defined below...